src/resolvers.ts

```diff
@@ -45,7 +45,7 @@ export const resolveContentPaths = (srcDir: string, nuxt = useNuxt()) => {
     })(),
 
     r(`${nuxt.options.dir.layouts}/**/*${sfcExtensions}`),
-    ...(nuxt.options.pages ? [r(`${nuxt.options.dir.pages}/**/*${sfcExtensions}`)] : []),
+    ...([true, undefined].includes(nuxt.options.pages) ? [r(`${nuxt.options.dir.pages}/**/*${sfcExtensions}`)] : []),
```

Comment:
Is this equivalent to `!!nuxt.options.pages`?

Reply:
No, `!!undefined` is false

Reply:
Ah you're right